### DISCUSSION TITLE
Is it possible to create grid layout?

### DISCUSSION THREAD

**Discussion by knight10-ux** [facebook/lexical#4372] [Ideas]
I am trying to build grid support and would like to develop plugin for it.

Main reason is to have ability to move paragraph blocks between cells to provide functionality for simple page builder. At the moment I need to create multiple editor instances and moving paragraph block around doesn't work in this case.

Do you think implementing this functionality would require changes in the core? Thank you for help!

**Comment by illodev** [comment]
I've actually developed a minimalist grid plugin that might be exactly what you're looking for. You can find it on GitHub at the following link: [Lexical Grid Plugin](https://github.com/capiloky/lexical-grid). If you're interested, feel free to check it out, and please don't hesitate to reach out if you have any questions or need assistance with it.

**Reply by T-N9** [reply]
Hello, 

I see that the Grid column node and Grid container node files are the same. There were invalid import from Grid container node.
